app/controllers/root_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class RootController < ApplicationController
-  def index; end
+  def index
+    @articles = Article.eager_load(:user).reorder(created_at: :DESC).all if user_signed_in?
```

Comment:
How do you like that fact that if in case if user is not signed in the `@articles` variable won't be defined at all. I think a better way is to give it a `[]` value